src/Sulu/Bundle/SearchBundle/DependencyInjection/Configuration.php

```diff
@@ -45,6 +45,16 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('website')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('additional_indexes')
```

Comment:
Not sure if I would go with `additional_indexes` by array. It will not allow to override things or prepend it in the bundles so I would go more with a configuration like this:

```yaml
website:
     indexes:
          pages: 'page_#webspace#_published' 
          products: 'my_products_published' 
          news: 'my_news_published' 
```

This make it to possible override the index or disable a prepended index by using set it to `null`.